packages/features/eventtypes/components/CreateEventTypeDialog.tsx

```diff
@@ -118,7 +118,7 @@ export default function CreateEventTypeDialog({
 
   const createMutation = trpc.viewer.eventTypes.create.useMutation({
     onSuccess: async ({ eventType }) => {
-      await utils.viewer.eventTypes.getByViewer.invalidate();
+      await utils.viewer.eventTypes.getEventTypesFromGroup.invalidate();
```

Comment:
This worked. Earlier newly created event wasn't showing in the list without refresh(when the flag is disabled)
```suggestion
       if (isInfiniteScrollEnabled) {
        await utils.viewer.eventTypes.getEventTypesFromGroup.invalidate();
      } else {
        await utils.viewer.eventTypes.getByViewer.invalidate();
      }
```